src/inc/clrconfigvalues.h

```diff
@@ -1027,6 +1027,8 @@ RETAIL_CONFIG_DWORD_INFO(EXTERNAL_ReadyToRun, W("ReadyToRun"), 0, "Enable/disabl
 
 #if defined(FEATURE_EVENT_TRACE) || defined(FEATURE_EVENTSOURCE_XPLAT)
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableEventLog, W("EnableEventLog"), 0, "Enable/disable use of EnableEventLogging mechanism ") // Off by default 
+RETAIL_CONFIG_STRING_INFO(EXTERNAL_EventSourceFilter, W("EventSourceFilter"), "")
```

Comment:
INTERNAL should be OK - the key is that you want it in retail builds which you get form RETAIL_CONFIG*.  INTERNAL just means that we don't document/expose it.